src/utils/map-style-utils/mapbox-gl-style-editor.js

```diff
@@ -115,13 +114,43 @@ export function getStyleDownloadUrl(styleUrl, accessToken, mapboxApiUrl) {
     const styleId = styleUrl.replace('mapbox://styles/', '');
 
     // https://api.mapbox.com/styles/v1/heshan0131/cjg1bfumo1cwm2rlrjxkinfgw?pluginName=Keplergl&access_token=<token>
-    return `${mapboxApiUrl || defaultMapboxApiUrl}/styles/v1/${styleId}?pluginName=Keplergl&access_token=${accessToken}`
+    return `${mapboxApiUrl || DEFAULT_MAPBOX_API_URL}/styles/v1/${styleId}?pluginName=Keplergl&access_token=${accessToken}`
```

Comment:
you can use the default value as part of the function signature
```
getStyleDownloadUrl(styleUrl, accessToken, mapboxApiUrl = DEFAULT_MAPBOX_API_URL)
```
same as you did in getStyleImageIcon